specification/communication/data-plane/Microsoft.CommunicationServicesChat/preview/2020-11-01-preview3/communicationserviceschat.json

```diff
@@ -0,0 +1,1589 @@
+            "x-ms-error-response": true
+          },
+          "503": {
+            "description": "Service unavailable",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Get chat thread": {
+            "$ref": "./examples/Threads_GetChatThread.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Threads"
+        ],
+        "summary": "Deletes a thread.",
+        "operationId": "DeleteChatThread",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "chatThreadId",
+            "description": "Thread id to delete.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Request successful."
+          },
+          "401": {
+            "description": "Unauthorized",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "403": {
+            "description": "Forbidden",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "429": {
+            "description": "Too many requests",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          },
+          "503": {
+            "description": "Service unavailable",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            },
+            "x-ms-error-response": true
+          }
+        },
+        "x-ms-examples": {
+          "Delete chat thread": {
+            "$ref": "./examples/Threads_DeleteChatThread.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ChatMessageReadReceipt": {
+      "description": "A chat message read receipt indicates the time a chat message was read by a recipient.",
+      "type": "object",
+      "properties": {
+        "senderId": {
+          "description": "Chat message read receipt sender id.",
+          "type": "string",
+          "readOnly": true,
+          "example": "8:acs:8540c0de-899f-5cce-acb5-3ec493af3800_0e59221d-0c1d-46ae-9544-c963ce56c10b"
+        },
+        "chatMessageId": {
+          "description": "Id for the chat message that has been read. This id is generated by the server.",
+          "type": "string",
+          "readOnly": true,
+          "example": "1591137790240"
+        },
+        "readOn": {
+          "format": "date-time",
+          "description": "Chat message read receipt timestamp. The timestamp is in ISO8601 format: `yyyy-MM-ddTHH:mm:ssZ`.",
```

Comment:
"The time at which the message was read....."

Reply:
updated